server/middleware/populateCurrentUser.ts

```diff
@@ -20,3 +20,41 @@ export default function populateCurrentUser(userService: UserService): RequestHa
     }
   }
 }
+
+export function getUserLocations(userService: UserService): RequestHandler {
```

Comment:
Naming should be consistent with CaseLoad being used in place of Location

Comment applies across multiple files/lines.